lambda-http4s/src/main/scala/feral/lambda/http4s/ApiGatewayProxyHandler.scala

```diff
@@ -80,9 +78,7 @@ object ApiGatewayProxyHandler {
       event: ApiGatewayProxyEventV2): F[Request[F]] = for {
     method <- Method.fromString(event.requestContext.http.method).liftTo[F]
     uri <- Uri.fromString(event.rawPath + "?" + event.rawQueryString).liftTo[F]
-    cookies = Some(event.cookies)
-      .filter(_.nonEmpty)
-      .map(Cookie.name.toString -> _.mkString("; "))
+    cookies = event.cookies.filter(_.nonEmpty).map(Cookie.name.toString -> _.mkString("; "))
```

Comment:
Hum, obviously not your fault but I can't remember why the code is duplicated here. Seems like this method was extracted out but then forgot to use it 😅

Reply:
Yes I was wondering too 😅 So I can remove the duplication from the apply method and use the extracted method? (Or remove `decodeEvent`, but it is useful for testing)

Reply:
Yes please, I think the intention was to use `decodeEvent` in `apply` :)